Changes

```diff
@@ -1,6 +1,59 @@
 OCaml 4.07
 ----------
 
+Highlights of this release include:
+
+- The way the standard library modules are organized internally has
+  changed (GPR#1010):
+  1. the List module (for example) is now named Stdlib__list
+  2. a new Stdlib module contains a series of aliases
+  such as (module List = Stdlib__list)
```

Comment:
~Why's this bit in brackets?~ Should the left paren. be before "such"?

Reply:
I used these parentheses to delimit user code (from prose), rather than parenthesizing. I could use backquotes here, but I'm hesitant to go towards full Markdown syntax.
(For now I will leave this part as such)

Reply:
Well, `(for example)` is not user code !  Just go with backticks, it's less confusing.